src/index.html

```diff
@@ -11,7 +11,109 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+          onsubmit="onSubmit()"  autocomplete="off">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname" class="form-field">
+          Surname:
+          <input type="text" name="surname"
+          id="surname" required>
```

Comment:
fix attributes indention in all cases
```suggestion
          <input
            type="text"
            name="surname"
            id="surname" required
          >
```